src/Dialogs/ForegroundProcessDialog.vala

```diff
@@ -44,6 +44,20 @@ public class Terminal.ForegroundProcessDialog : Granite.MessageDialog {
         );
     }
 
+    public ForegroundProcessDialog.kill_process (MainWindow parent) {
+        Object (
+            transient_for: parent,
+            primary_text: _("Are you sure you want to kill this process?"),
+            secondary_text:
+                _("There is an active process on this terminal.") + " " +
+                _("Are you sure you want to kill it?"),
+            buttons: Gtk.ButtonsType.CANCEL,
+            button_label: _("Kill Process")
```

Comment:
I feel like we should avoid the word "Kill". "Terminate" is probably better language

Reply:
@danrabbit even "Terminate" seems a little… excessive. Couldn't it just be “End”?

Reply:
There are technical differences between "Kill" and "End / Terminate" although we may not need trouble the user with them.  I presume the intended use of the option would be to forcibly end (kill) a process that could not be ended normally.